client/middleware/error-404.js

```diff
@@ -1,12 +1,24 @@
+var Templates = require('../templates');
+
 module.exports = function (page) {
-  page(function (ctx, next) {
+  // page(function (ctx, next) {
+  //   next();
+  // });
+  page(render);
+  function render (ctx, next) {
     console.warn('Page not found', ctx.path);
```

Comment:
I think we use console.war only to display which file couldn't be loaded and when a document of elasticsaerch doesn't belong to any of the main type (people, documents or object)